types/google.visualization/index.d.ts

```diff
@@ -1724,6 +1731,61 @@ declare namespace google {
             yellowTo?: number;
         }
 
+        //#endregion
+        //#region Sankey
+
+        // https://developers.google.com/chart/interactive/docs/gallery/sankey
+        export class Sankey extends ChartBaseClearable {
+            draw(data: DataTable | DataView, options: SankeyChartOptions): void;
+            getBoundingBox(id: string): ChartBoundingBox;
```

Comment:
The documentation is wrong here, the `getBoundingBox` does not actually exist at runtime